Sources/Private/MainThread/LayerContainers/Utility/CachedImageProvider.swift

```diff
@@ -0,0 +1,51 @@
+// Created by Jianjun Wu on 2022/5/12.
+// Copyright © 2022 Airbnb Inc. All rights reserved.
+
+import CoreGraphics
+import Foundation
+#if os(iOS) || os(tvOS) || os(watchOS) || targetEnvironment(macCatalyst)
+import UIKit
```

Comment:
What is the UIKit import for?

Reply:
Thanks for point out!
I once used cache UIImage to cache, and forgot to remove this after I changed to use CGImage.